data/json/professions.json

```diff
@@ -1726,7 +1726,7 @@
         ],
         "entries": [
           { "item": "glock_18c", "ammo-item": "9mmP", "container-item": "holster" },
-          { "item": "magbandolier", "contents-group": "army_mags_glock17" }
+          { "item": "tacvest", "container-group": "army_mags_glock17" }
```

Comment:
Given you changed this line but not the other at 1891 from ``contents-group`` to ``container-group``, one must be wrong.

Reply:
Odd. Not sure how that happened.

Reply:
Um, you not gonna change it or the other one?

Reply:
Forgot to push.